.gitignore

```diff
@@ -5,6 +5,17 @@
 /tmp
 /out-tsc
 /release-builds
+/linux-builds
+/darwin-builds
+/macos-builds
+/win32-builds
+/linux-packages
+/darwin-packages
+/windows-packages
+/all-packages
+/mac
+/windows
+/linux
```

Comment:
It looks like not all the directories listed here are actually created. Only directories that are created by the build should be listed here.

In general, if it's possible it would be best to keep all the release builds in a single place, under a top-level directory. E.g. `apps/mac-builds` or `apps/mac`, `apps/linux-builds` etc, to avoid creating many directories at the root level.

Reply:
Ok, will fix it as suggested.